app/importers/rows/attendance_row.rb

```diff
@@ -33,6 +35,13 @@ def build
       student_id: student.try(:id),
     )
 
+    attendance_event.assign_attributes(
```

Comment:
One thing that might be good is to add a `PerDistrict` method to indicate whether we expect to be finding all these fields, since that seems different across districts.  That way it's clear in the code that "this is intentionally only for Somerville right now" rather than not being able to tell until looking at the data.